Metro-Records/la-metro-councilmatic
# A11y Improvements - Understandable

## Overview

This branch aims to improve on the "understandable" category of WCAG standards for the app. Each of the areas of improvement are listed in the connected issue.

This is a smaller set of changes which are all invisible outside of using a screen reader and/or a validator like lighthouse. They are focused around adding labels and explicitly setting the language for text in spanish with the `lang` attribute.

- Connects #1023 

### Notes

The original issues for each category are listed in #1023

- 3.3.2 Labels or Instructions:
  - For the search bar, I instead went with adding in an `aria-label` and an `aria-describedby` combo for the search bar, where the descriptions were id’d with `search-help`.
  - The only input I noticed had a missing label that I had trouble inserting was the on the select2 input.
- 3.2.4 Consistent Identification:
  - Accessible labels for the right chevrons has already been taken care of in https://github.com/Metro-Records/la-metro-councilmatic/commit/b26fc0b7e11439c4c14b02f3d19dcd309f4c94f6.

## Testing Instructions

 * Get to the site
 * Asses if the points covered in the issue have been properly addressed.
   * Use lighthouse or another a11y tool to check for labels and distinct id's throughout various pages
   * Use a screen reader like VoiceOver to check that everything is being read as expected, including the Spanish aria-label text on the "Ver en Español" buttons
     * Note: To get Spanish in VoiceOver go to Settings > Accessibility > Open VoiceOver Utility > Speech and click the plus button to add a language.
